subprojects/matrix-benchmarking-workloads/rhods-notebooks-ux/horreum_lts_store.py

```diff
@@ -20,85 +20,96 @@ def register_lts_metric(cluster_role, metric):
     for (name, query) in metric.items():
         lts_metrics[cluster_role].append((name, query))
 
+
 def _parse_lts_dir(add_to_matrix, dirname, import_settings):
-    with open(f'{dirname}/data.json') as f:
+    with open(dirname / "data.json") as f:
         data = json.load(f)
-        add_to_matrix(_encode_json(data, dirname, data['settings']))
+
+    add_to_matrix(_encode_json(data, dirname, data["settings"]))
```

Comment:
moved `add_to_matrix` out of the `with` block, as it's unrelated to the file operation